### ISSUE TITLE
'Notation preview' mode in percussion panel should read out notation details in speech output

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32788]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

In the Percussion Panel the options for Pad Names or Notation Preview both give the same speech output from the screen reader.  It is difficult for a blind user to find out how MuseScore is notating each instrument and so this would be an ideal way to be able to know how each drum looks.

The current info of instrument name, shortcut key and midi key is great when Pad Name is checked, but I would have expected when Notation Preview is checked to then hear instrument name, line/space it is notated on, stem direction and note head shape.

STEPS:
1.  Create a score with a drum kit and navigate to that staff.  Press Tab several times till you get to Percussion and then to one of the Drum Pads.
2. Note the speech output such as "Bass drum 1, not selected, shortcut A, Midi C2, Row 1 column 1.".  NB the not selected here seems erroneous as pressing enter either previews the sound or adds it to the score depending on setting, but can it actually be selected?
3. Tab back to the layout button and press enter.
4. Arrow down till you hear "Notation Preview not checked" (If checked go to Drum Pad Names not checked instead) and press enter.
5. Press Tab and arrows to get back to the same pad as before.  Note the speech is identical. "Bass drum 1, not selected, shortcut A, Midi C2, Row 1 column 1.".  

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7 beta

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
